src/drivers/imx/sdma.c

```diff
@@ -95,6 +95,11 @@ static void sdma_enable_channel(struct dma *dma, int channel)
 	dma_reg_update_bits(dma, SDMA_HSTART, BIT(channel), BIT(channel));
 }
 
+static void sdma_disable_channel(struct dma *dma, int channel)
+{
+	dma_reg_update_bits(dma, SDMA_STOP_STAT, BIT(channel), BIT(channel));
```

Comment:
Accidentally stop multiple channels? Same explanation as previous commit, you are stopping ALL active channels except those that have a second "start" command given to them.

Reply:
@paulstelian97 I think you are right. Writing a 1 will stop a started channel. Will update. Same for enable_channel.